README.md

```diff
@@ -233,6 +237,42 @@ http {
 }   
 ```
 
+## Disabled Cache Configurations
+
+It is recommended to cache the results of the call to the Curity Identity Server so that you avoid triggering an introspection request for every API request. If you wish to disable caching you should extend the default [proxy_buffer_size](https://nginx.org/en/docs/http/ngx_http_proxy_module.html#directives) to ensure that the module can read large JWTs. Do so by updating the configuration of the introspection request as in the following example.
```

Comment:
Explain what a non caching configuration looks like and how you increase the default [proxy-buffer-size](https://nginx.org/en/docs/http/ngx_http_proxy_module.html#proxy_buffering) when there are large JWTs:

- When buffering is enabled, nginx receives a response from the proxied server as soon as possible, saving it into the buffers set by the proxy_buffer_size and proxy_buffers directives
- When buffering is disabled, the maximum size of the data that nginx can receive from the server at a time is set by the proxy_buffer_size directive.